examples/Scenario1Bundle-jws-payload-expanded.json

```diff
@@ -1,10 +1,11 @@
 {
   "iss": "https://spec.smarthealth.cards/examples/issuer",
-  "nbf": 1628269596,
+  "nbf": 1629150293,
   "vc": {
     "type": [
       "https://smarthealth.cards#health-card",
-      "https://smarthealth.cards#labresult"
+      "https://smarthealth.cards#covid19",
+      "https://smarthealth.cards#immunization"
     ],
     "credentialSubject": {
       "fhirVersion": null,
```

Comment:
Should we also add a valid fhirVersion as a part of this PR?

Reply:
(May not be necessary to do manually here once we can make these updates using the inheritance code in https://github.com/dvci/health_cards/pull/87)

Reply:
Added fhir_version.